miniconda/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Travis Swicegood
 
 RUN apt-get update && apt-get install -y wget bzip2
 RUN echo 'export PATH=/opt/conda/bin:$PATH' > /etc/profile.d/conda.sh && \
-    wget --quiet http://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh && \
+    wget --quiet https://repo.continuum.io/miniconda/Miniconda-3.9.1-Linux-x86_64.sh && \
```

Comment:
@asmeurer It could work, it's just not installed.  The base image this is built off of has next to nothing in it… which gives me my answer I think.  Trying to re-build with `ca-certificates` installed.

Re-using `curl`: I have some part of my brain that's saying `wget` is needed elsewhere in the stack, but I can't place it.  Might be I needed it for other install scripts when I wrote the first versions of this.